docs/changes/3.0.0.rst

```diff
@@ -14,6 +14,57 @@ Changes to Supported Platforms
 
 .. towncrier release notes start
 
+
```

Comment:
As I understand you only included the rendered output to illustrate the proposed changes while the review is in draft.

We'll have to remove it from the final PR.